.github/workflows/image.yml

```diff
@@ -0,0 +1,51 @@
+# This workflow will build images for the codeflare-operator, then push the images to the `project-codeflare` quay repo
+
+name: Image
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Grab New Version Tag
+      id: tag_name
+      run: |
+        echo ::set-output name=SOURCE_TAG::${GITHUB_REF#refs/tags/}
+
+    - name: Login to Quay.io
+      uses: redhat-actions/podman-login@v1
+      with:
+        username: ${{ secrets.QUAY_ID }}
+        password: ${{ secrets.QUAY_TOKEN }}
+        registry: quay.io
+
+    - name: Login to Red Hat Registry
+      uses: redhat-actions/podman-login@v1
+      with:
+        username: ${{ secrets.RH_REG_ID }}
+        password: ${{ secrets.RH_REG_TOKEN }}
+        registry: registry.redhat.io
+
+    - name: Image Build
+      run: |
+        make image-build -e IMG=quay.io/project-codeflare/codeflare-operator:${SOURCE_TAG}
+        podman tag quay.io/project-codeflare/codeflare-operator:${SOURCE_TAG} quay.io/project-codeflare/codeflare-operator:latest
+        make bundle-build -e BUNDLE_IMG=quay.io/project-codeflare/codeflare-operator-bundle:${SOURCE_TAG}
```

Comment:
does `make bundle` need to be called? From what I can tell, this will just build an image with whatever bundle files already exist on the repo.

Reply:
Oh hmm, yeah good catch